src/scene/scene.cpp

```diff
@@ -848,69 +847,112 @@ scripting::LuaLibrary Scene::luaLibrary() {
+
+If you only want to change multiple property values simultaneously, also see the
+#setPropertyValue function. The `setPropertyValueSingle` function however will work more
+efficiently for individual property value.
+
+\\param uri The URI that identifies the property
+\\param value The new value to which the property identified by the `uri` should be
+changed to. The type of this parameter must agree with the type of the selected property
+\\param duration The number of seconds over which the change will occur. If not provided
+or the provided value is 0, the change is instantaneously.
+\\param easing If a duration larger than 0 is provided, this parameter controls the manner
+in which the parameter is interpolated. Has to be one of "Linear", "QuadraticEaseIn",
+"QuadraticEaseOut", "QuadraticEaseInOut", "CubicEaseIn", "CubicEaseOut", "CubicEaseInOut",
+"QuarticEaseIn", "QuarticEaseOut", "QuarticEaseInOut", "QuinticEaseIn", "QuinticEaseOut",
+"QuinticEaseInOut", "SineEaseIn", "SineEaseOut", "SineEaseInOut", "CircularEaseIn",
+"CircularEaseOut", "CircularEaseInOut", "ExponentialEaseIn", "ExponentialEaseOut",
+"ExponentialEaseInOut", "ElasticEaseIn", "ElasticEaseOut", "ElasticEaseInOut",
+"BounceEaseIn", "BounceEaseOut", "BounceEaseInOut"
+\\param postscript This parameter specifies a Lua script that will be executed once the
+change of property value is completed. If a duration larger than 0 was provided, it is
+at the end of the interpolation. If 0 was provided, the script runs immediately.
+)",
+                { "src/scene/scene_lua.inl", 378 }
             },
             {
                 "getPropertyValue",
                 &luascriptfunctions::propertyGetValueDeprecated,
-                {},
-                "",
+                {
+                    { "uri", "String" }
+                },
+                "String | Number | Boolean | Table",
                 "Returns the value the property, identified by the provided URI. "
                 "Deprecated in favor of the 'propertyValue' function",
-                {}
+                { "src/scene/scene_lua.inl", 495 }
             },
             {
                 "propertyValue",
                 &luascriptfunctions::propertyGetValue,
-                {},
-                "",
-                "Returns the value the property, identified by the provided URI. "
-                "Deprecated in favor of the 'propertyValue' function",
-                {}
+                {
+                    { "uri", "String" }
+                },
+                "String | Number | Boolean | Table",
+                "Returns the value the property, identified by the provided URI. This "
+                "function will provide an error message if no property matching the URI "
+                "is found.",
+                { "src/scene/scene_lua.inl", 495 }
             },
             codegen::lua::HasProperty,
             codegen::lua::PropertyDeprecated,
```

Comment:
Can't access the code, but the tag and wildcard behavior needs to be added to the `property` function as well. 

And maybe removeSceneGraphNodesFromRegex``?  But seems like that one does not support the tags...?

Reply:
The `propertyValue` function doesn't support wildcards and will only ever match a single property value.

The `removeSceneGraphNodesFromRegex` is used in the assetvalidation script afaik, so we'll need that

Reply:
Gotcha!  I'll add the tags into that function if I get the time over the weekend

Reply:
Not sure it's a big deal if it doesn't! Especially since it's so concretely named to explicitly use regex